frontends/krita/krita_comfy/widgets/prompt.py

```diff
@@ -1,5 +1,27 @@
+import re
+from PyQt5.QtGui import QSyntaxHighlighter, QColor, QColorConstants
```

Comment:
This breaks my plugin, can't import `QColorConstants` on my end. Seems PyQt version is lower than required.

Reply:
Ok let me look at another way to do it

Reply:
@JasonS09 can you try it now?